mne/preprocessing/infomax_.py

```diff
@@ -136,7 +148,8 @@ def infomax(data, weights=None, l_rate=None, block=None, w_change=1e-12,
     while step < max_iter:
 
         # shuffle data at each step
-        rng.seed(step)  # --> permutation is fixed but differs at each step
```

Comment:
I agree. It seems actually a bit odd to me to reset the seed to the to the value of the iteration count. I never understood that line. The random state should be set at the begninning such that it starts at a fixed position. @kazemakase @jdammers Is this fixed resetting part of the orginal algo?